docs/usage/flower17/flower17_scikit.rst

```diff
@@ -0,0 +1,108 @@
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **convert_color**  |  This operation uses the OpenCV function `cv2.cvtColor` and converts the image from one colorspace to another.                                                                            |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **calcHist**       |  This operation extracts the histogram from the image array and is used in this tutorial to quantify the colors present in a flower image.                                                |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **normalize**      |  This operation will be used to normalize the value range of the histogram here.                                                                                                          |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **HuMoments**      |  This operation calculates the seven Hu invariants which quantifies the information regarding the orientation of the image. `Image Moments <https://en.wikipedia.org/wiki/Image_moment>`_ |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **Haralick**       |  This operation extracts the texture features from the image.                                                                                                                             |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|  **flatten**        |  This operation returns a copy of numpy array collapsed into 1 dimension.                                                                                                                 |
++---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+The dataflow visualised:
+
+.. image:: /../examples/flower17/sklearn-opencv/dataflow_diagram.svg
```

Comment:
Why are we only flattening normalized calcHist output?

Reply:
The other 2 feature extractors are 1 dimensional so they don't need to be flattened.